test/TestApps/ToastNotificationsDemoApp/main.cpp

```diff
@@ -196,11 +238,15 @@ int main()
     std::wcout << L"Requesting PushNotificationChannel...\n\n";
     winrt::PushNotificationChannel channel{ RequestChannel() };
 
+    std::wcout << L"Post a Toast..." << std::endl;
+    PostToastHelper(L"Tag", L"Group");
+    std::wcout << L"Done.\n\n";
+
     std::wcout << L"Press enter to exit the app.\n\n";
     std::cin.ignore();
 
     // If you want to stop receiving AppNotifications for the app
```

Comment:
Update comment here: "Call Unregister so that COM can launch a new process for ToastInvokes after we terminate this process.